omimIssues.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
If there are genuine issues with the latest release of `omim.ttl`, and I'm reporting them, I might as well commit the report as well. But I can always add this to the `.gitignore`.

Reply:
I reported on these issues and no longer need this. Removed.